depends/packages/gmp.mk

```diff
@@ -1,13 +1,14 @@
 package=gmp
-$(package)_version=6.2.1
-$(package)_download_path=https://gmplib.org/download/gmp
+$(package)_version=6.3.0
+$(package)_download_path=https://ftp.gnu.org/gnu/gmp
 $(package)_file_name=gmp-$($(package)_version).tar.bz2
-$(package)_sha256_hash=eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
+$(package)_sha256_hash=ac28211a7cfb609bae2e2c8d6058d66c8fe96434f740cf6fe2e47b000d1c20cb
 
 define $(package)_set_vars
-$(package)_config_opts+=--enable-cxx --enable-fat --disable-shared
-$(package)_cflags_armv7l_linux+=-march=armv7-a
-$(package)_cflags_aarch64_darwin+=-march=armv8-a
+$(package)_config_opts += --disable-shared --enable-cxx --enable-fat
+$(package)_cflags_aarch64 += -march=armv8-a
+$(package)_cflags_armv7l += -march=armv7-a
+$(package)_cflags_x86_64 += -march=x86-64
```

Comment:
has any thought been had to risc-v?

Reply:
It's unclear what would be a suitable "base" `march` for 64-bit RISC-V since there seems to be limited use of `riscv64` as such a parameter.

`rv64i` would be making the fewest possible assumptions (the `E` variant is supported only assembly tools by LLVM as of this writing, [source](https://llvm.org/docs/RISCVUsage.html), from here on out, let's assume the `I` variant) but on the other hand, there is documentation that asserts that `rv64g` is complete for general computing ([source](https://five-embeddev.com/riscv-user-isa-manual/Priv-v1.12/gmaps.html)) and projects like Debian use `rv64gc` as the baseline ([source](https://wiki.debian.org/RISC-V#Debian_port_information)) though vector extensions require `rv64gcv`.

Extensions are enumerated in the `march` explicitly, of which there are many ([source](https://gcc.gnu.org/onlinedocs/gcc/RISC-V-Options.html)) though LLVM offers 6 different profiles (see source above) based on the architecture profiles specified by the RISC-V foundation ([source](https://github.com/riscv/riscv-profiles/blob/main/src/profiles.adoc)).

---

As an aside, the `x86-64` `march` is a base profile for AMD64 but we _could_ investigate if we should enforce minimum processor requirements in our AMD64 binaries _if_ benchmarks prove provide significant performance over the base profile being used now but that's an exercise for a future PR.